src/test/unit/common/models/LastUpdateSectionBuilder/lastUpdateSectionBuilder.test.ts

```diff
@@ -0,0 +1,72 @@
+import {LastUpdateSectionBuilder} from 'models/LastUpdateSectionBuilder/latestUpdateBuilder';
+
+describe('LatestUpdateTest tests', ()=> {
+  it('should create title', ()=> {
+    //Given
+    const title = 'testTitle';
+    const titleObject = new LastUpdateSectionBuilder()
+      .addTitle(title)
+      .build();
+
+    //When
+    const result = new LastUpdateSectionBuilder()
+      .addTitle(title)
+      .build();
+    //When Then
```

Comment:
```suggestion
    //Then
```
Applies to the other tests here too

Reply:
done!